openam-server-only/pom.xml

```diff
@@ -443,7 +443,7 @@
                                 <transformationSet>
                                     <dir>${project.build.outputDirectory}</dir>
                                     <includes>
-                                        <include>amAuth.xml</include>
+                                        <include>serviceNames.xml</include>
```

Comment:
Not much really. Actually this change probably is erroneous: the XML transformation is done to ensure that if OpenAM was compiled with SecurID included, then amAuth.xml will include the SecurID auth module in the authenticators list.
serviceNames.xml file doesn't even exist... There is only serviceNames.properties file, but that is just resource filtered by Maven. (serviceNames.properties needs the filtering to make sure that amAuthSecurID.xml service gets loaded when AM was built with SecurID included.)